oaknational/Oak-Web-Application
# feat(refactor): refactor tracking events into respective views

## Description
Have moved the tracking functions for views into the the relevant views; this includes
intro
video
quiz
worksheet downloads

for activity started, abandoned and completed

Music year: 1997

- List of changes

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2860--oak-web-application.netlify.thenational.academy
2. Click on a lesson; complete a lesson
3. You should see an avo event fired for every action taken accross the lesson 

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
